spec/services/amr/csv_parser_and_upserter_spec.rb

```diff
@@ -0,0 +1,293 @@
+require 'rails_helper'
+require 'fileutils'
+require 'fakefs/spec_helpers'
+
+module Amr
+  describe CsvParserAndUpserter do
```

Comment:
THis is the old csv_importer spec - renamed - at the moment it's testing the underlying services, doing parsing and upserting - ideally at some point would be good to split those out (when a new test is required?)